cms/articles/tests/test_models.py

```diff
@@ -177,6 +177,50 @@ def test_next_date_must_be_after_release_date(self):
 
         self.assertListEqual(info.exception.messages, ["The next release date must be after the release date."])
 
+    def test_clean_validates_a_min_of_two_headline_figures_are_added(self):
```

Comment:
Could also do with a test for > 6 headline figures just to be safe.

Reply:
that is core functionality, but sure

Reply:
9741e449379c05e14ceffa29c51bcf3fb57aaba8